src/odm_sharing/tools/share.py

```diff
@@ -189,7 +192,7 @@ def share(
         table_filter = get_tables(org_queries)
     except rules.ParseError:
         # XXX: error messages are already printed at this point
-        quit()
+        return
```

Comment:
Not sure what you mean by a nested exception?

Reply:
sys.exit raises SystemExit. If I call that from within an except clause, then it causes a long list of exceptions like: "this exception was raised from that exception, blabla"